app/models/crop.rb

```diff
@@ -38,7 +38,11 @@ def search_data
   end
 
   def main_image_path
-    pictures.first.attachment.url if pictures.present?
+    if pictures.present?
+      pictures.first.attachment.url
+    else
+      ActionController::Base.helpers.asset_path('baren_field.jpg')
```

Comment:
It's a violation of MVC to reference ActionController inside of a model. In this case, you need to use PaperClip to generate the URL rather than ActionController. It's going to be a little bit more complicated in this situation, because the image is an association rather than a field.

I will pull down your branch and take care of this after I poke around that CSV stuff this morning.